tests/x11/multi_users_dm.pm

```diff
@@ -21,6 +21,9 @@ use utils;
 
 sub ensure_multi_user_target {
     type_string "systemctl isolate multi-user.target\n";
+    wait_still_screen 5;
+    send_key "ctrl-alt-f" . get_root_console_tty;
```

Comment:
Let's at least wrap it in wait_screen_change, because we can send alt-c too early.

Reply:
`wait_screen_change` doesn't seem to be triggered in this case:
http://artemis.suse.de/tests/166
```patch
+    wait_screen_change {
+        type_string "systemctl isolate multi-user.target\n";
+        send_key "ctrl-alt-f" . get_root_console_tty;
+    }
```

Reply:
I actually meant wrapping send_key. In case we type text it doesn't make sense as screen changes after first symbol. Could you please try with just send_key?

Reply:
Updated the commit and tested: http://artemis.suse.de/tests/167

Reply:
Ok, now we get exact glitches I got while working on this issue. I also tried to restart X by restarting services, it didn't help. So I guess we should consider rebooting in the test as there is no progress on the bug...

Reply:
Oh, sorry for this confusion, but I meant send key on L#25 mainly, so we switch to tty before sending ctrl-c. But wrapped ctrl-c also doesn't hurt.